public_html/wp-content/plugins/wcpt/views/applications/events/shortcode-application.php

```diff
@@ -0,0 +1,504 @@
+								</li>
+								<li>
+									<input type="checkbox" name="q_describe_event[]" id="q_1908778"
+										   value="Topic focused"/>
+
+									<label for="q_1908778">Topic focused (design, development, security, accessibility, SEO, agencies, marketing, etc.)</label>
+								</li>
+								<li>
+									<input type="checkbox" name="q_describe_event[]" id="q_1908779"
+										   value="Identity focused"/>
+
+									<label for="q_1908779">Identity focused (WordPress for Students, WordPress for Women, BlackPress, etc.)</label>
+								</li>
+								<li>
+									<input type="checkbox" name="q_describe_event[]" id="q_1908831"
+										   value="Job-status focused"/>
+
+									<label for="q_1908831">Job-status focused (job seekers, freelancers, business owners, etc)</label>
+								</li>
+								<li>
+									<input type="checkbox" name="q_describe_event[]" id="q_1908832"
+										   value="For a specific expertise level"/>
+
+									<label for="q_1908832">For a specific expertise level (beginners, intermediate, advanced users)</label>
+								</li>
+								<li>
+									<input type="checkbox" name="q_describe_event[]" id="q_1908833"
+										   value="Sustainable event"/>
+
+									<label for="q_1908833">Sustainable event (no swag, small/easy to organize, low budget, etc.)</label>
+								</li>
+								<li>
+									<label for="q_other">Other:</label>
+
+									<input type="text" class="other" name="q_describe_event_other"
+										   id="q_other" value="" style="width: 75%"/>
+								</li>
+							</ul>
+						</div>
+					</div>
+				</div>
+
+
+				<div class="PDF_question" id="pd-question-16">
+					<div class="qContent">
+						<div class="qText">
+							Approximate number of attendees:
+							<span class="PDF_mand">*</span>
+						</div>
+
+						<div class="PDF_QT100">
+							<input value="" maxlength="500" name="q_how_many_attendees" class="large required"
+								   type="text" title="Approximate number of attendees" required/>
+						</div>
+					</div>
+				</div>
+
+				<div class="PDF_question" id="pd-question-17">
+					<div class="qContent">
+						<div class="qText">
+							If you have co-organizers, please share their name(s), email address(es) and WordPress.org username(s)
+						</div>
+
+						<div class="PDF_QT200">
+							<textarea name="q_co_organizer_contact_info" class="small" rows="10" cols="40"
+									  title="If you have co-organizers, please share their name(s), email address(es) and WordPress.org username(s)"></textarea>
+						</div>
+					</div>
+				</div>
+
+				<div class="PDF_question" id="pd-question-18">
+					<div class="qContent">
+						<div class="qText">
+							Your event URL will be: https://events.wordpress.org/[location]/[year]/[event-name]
+							e.g. https://events.wordpress.org/narnia/2024/communityday
+							What would you like to be the [event-name] at the end of your event URL?
```

Comment:
At the very least "What would ..." should be on its own line. 

It's very wordy though, do we need to pass this effort along to the user in this manner? @hellojuliarose